commonv5/ts/sonarqube/Analysis.ts

```diff
@@ -86,40 +107,33 @@ export default class Analysis {
 
       const threshold =
         condition.status === "WARN" ? condition.warningThreshold : condition.errorThreshold;
-      return `<tr>
-        <td><span style="padding-right:4px;">${metric.name}</span></td>
-        <td style="text-align:center; color:#fff; background-color:${this.getQualityGateColor()};">
-          <span style="padding:0px 4px; line-height:20px;">${formatMeasure(
-            condition.actualValue,
-            metric.type,
-          )}</span>
-        </td>
-        <td>
-          <span style="padding-left:4px">${formatMeasure(condition.comparator, "COMPARATOR")}</span>
-          <span style="padding-left:4px">${formatMeasure(threshold, metric.type)}</span>
-        </td>
-      </tr>`;
+      return `<li style="list-style: none; margin-top: 13px;">
+        <span>${metric.name}</span>
+        <span style="${measureStyle(this.getQualityGateColor())}">
+          ${formatMeasure(condition.actualValue, metric.type)}</span>
+        <span>(required ${
+          metric.type !== "RATING" ? formatMeasure(condition.comparator, "COMPARATOR") + " " : ""
+        }${formatMeasure(threshold, metric.type)})</span>
+      </li>`;
     });
 
-    const tableStyle = `
-      margin-top: 8px;
-      border-top: 1px solid #eee;
-      border-collapse: separate;
-      border-spacing: 0 4px;
-    `;
-    return `<table border="0" style="${tableStyle}">
-      <tbody>
+    return `<ul style="${textStyle()}">
         ${rows.join(" \r\n").trim()}
-      </tbody>
-    </table>\r\n\r\n`; // 2 carriage returns to prevent any malformed summary results
+    </ul>\r\n\r\n`; // 2 carriage returns to prevent any malformed summary results
   }
 
   private getDashboardLink() {
     if (!this.dashboardUrl) {
       return "";
     }
-    const linkText = `Detailed ${this.endpointType} report &gt;`;
-    return `[${linkText}](${this.dashboardUrl})`;
+    const linkText = `Detailed ${this.endpointType} report`;
+
+    return `<div style="margin-top: 16px;">
+        <a href="${this.dashboardUrl}" style="${textStyle("#0067B8")}">
+        ${linkText}
+        <span class="bowtie-icon bowtie-navigate-external" style="color: #0067B8; font-size: 20px; vertical-align: middle;"></span>
```

Comment:
I am a bit surprised that we do not use the lighter color for the link, but I did see that on figma it is like this, so I guess that's expected

Reply:
Good point, I double checked with Simo and we agreed on using color from DS, same for text color